modules/acquia_cms_search/tests/src/Functional/AcquiaSearchIntegrationFromTourPageTest.php

```diff
@@ -51,10 +50,6 @@ class AcquiaSearchIntegrationFromTourPageTest extends BrowserTestBase {
    * {@inheritdoc}
    */
   protected function setUp(): void {
-    // Check active subscription of acquia environment.
-    if (!AcquiaDrupalEnvironmentDetector::getAhApplicationUuid()) {
-      $this->markTestSkipped('This test can only run when acquia application uuid is set.');
-    }
```

Comment:
do not remove this as if someone run test and ```AcquiaDrupalEnvironmentDetector::getAhApplicationUuid()``` return null or false tests will fail.

Reply:
Not only this, we need following three variable available with correct value in order to get test passing
`export CONNECTOR_ID=some-value
export SEARCH_UUID=some-value
export CONNECTOR_KEY=some-value`

Also, checking `AH_APPLICATION_UUID` valid.

Reply:
got it.